python.d/w1sensor.chart.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+# Description: 1-wire temperature monitor netdata python.d module
+# Author: Diomidis Spinellis <http://www.spinellis.gr>
+
+import os
+import re
+from bases.FrameworkServices.SimpleService import SimpleService
+
+# default module values (can be overridden per job in `config`)
+update_every = 5
+priority = 60000
+retries = 60
```

Comment:
Unless I'm mistaken, you're using the same number of retries as the default, in which case this line can probably be removed.

Reply:
Yes, [retries and priority](https://github.com/firehol/netdata/blob/master/plugins.d/python.d.plugin#L38-L39) can be removed since it is current default

Reply:
Thank you; I removed them.